src/main/java/org/broadinstitute/consent/http/models/dto/DataSetDTO.java

```diff
@@ -155,7 +156,7 @@ public Integer getDataSetId() {
     }
 
     public void setAlias(Integer alias) {
-        this.alias = DatasetUtil.parseAlias(alias);
+        this.alias = "DUOS-" + StringUtils.leftPad(alias.toString(), 6, "0");
     }
```

Comment:
We should be referencing the Dataset class here if we need it. One way to do that is to make it a public static method there so we don't have to copy the guts of that method in different places.